scripts/check_formatting

```diff
@@ -20,3 +20,5 @@ find scripts -type f -exec shellcheck {} +
 # Fortunately, rustfmt has the --check option that will make it exit with 1
 # if formatting has to be applied.
 find examples rust -type f -name '*.rs' -exec rustfmt --check {} +
+
+prettier --check ./*.md ./docs/*.md
```

Comment:
Might be worth doing a `find` under `docs/` to catch any future subdirectories, even though I guess it will end up needing two invocations:
```bash
prettier --check *.md
find docs -name '*.md' -exec prettier --check {} +
```

Reply:
PTAL.